frontend/nc-games/src/App.js

```diff
@@ -1,23 +1,30 @@
 import './App.css';
-
+import { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import * as api from "./api"
+import Welcome from "./components/Welcome";
+import Reviewlist from "./components/Reviewlist";
+//import Categories from "./components/Categories";
 function App() {
+  
+  const [user, setUser] = useState('user');
+  const [reviews, setReviews] = useState([]);
+  const [categories, setCategories] = useState([]);
+
+  useEffect(()=>{ 
+
+  }, []);
+
+
   return (
+    <BrowserRouter>
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+          <Routes>
+            <Route path="/Welcome" element={<Welcome />} />
+            <Route path="/reviewList" element={<Reviewlist />} />
```

Comment:
your routes should all be lowercase - your browser will ignore any uppercase letter anyway but it's good practice